### PR TITLE
setup uiowa08

### PR DESCRIPTION
**Pull request by joewhitsitt**
Relates to https://github.com/uiowa/uiowa/issues/7929
Following step 2 of the Code section: https://docs.prod.drupal.uiowa.edu/acquia-cloud

# How to test

- Compare to previous app setup PR: https://github.com/uiowa/uiowa/pull/5898
- Note that https://github.com/uiowa/uiowa/pull/7952/files#diff-0ba8f676e05070f29dc5fb883fb82eba43544fd76f7c3ca1efb68d3514880b50R22 is the same SVN as what is listed in https://github.com/uiowa/uiowa/blob/main/.travis.yml#L21 so no addition to that file is necessary.
- Compare drush file to current main version of ACN uiowa07 https://github.com/uiowa/uiowa/blob/main/drush/sites/uiowa07.site.yml

### CODE REVIEW

File: drush/sites/uiowa08.site.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  uri: uiowa.ddev.site
+dev:
+  uri: uiowa08dev.prod.acquia-sites.com
+  host: uiowa08dev.ssh.prod.acquia-sites.com
+  options: {  }
+  paths: { dump-dir: /mnt/tmp }
+  root: /var/www/html/uiowa08.dev/docroot
+  user: uiowa08.dev
+prod:
+  uri: uiowa08prod.prod.acquia-sites.com
```
Comment (e-marie-w) [inline drush/sites/uiowa08.site.yml:12]: Should this line and the next have that duplicate prod in it? Based on previous app setups it seems like it should be uiowa08.prod.acquia-sites.com

File: drush/sites/uiowa08.site.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  host: uiowa08prod.ssh.prod.acquia-sites.com
+  options: {  }
+  paths: { dump-dir: /mnt/tmp }
+  root: /var/www/html/uiowa08.prod/docroot
+  user: uiowa08.prod
+test:
+  uri: uiowa08stage.prod.acquia-sites.com
```
Comment (e-marie-w) [inline drush/sites/uiowa08.site.yml:19]: Previous app setups had stage abbreviated in this line and the next, to uiowa08stg.prod.acquia-sites.com.

File: drush/sites/uiowa08.site.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  host: uiowa08stage.ssh.prod.acquia-sites.com
+  options: {  }
+  paths: { dump-dir: /mnt/tmp }
+  root: /var/www/html/uiowa08.stage/docroot
+  user: uiowa08.stage
```
Comment (e-marie-w) [inline drush/sites/uiowa08.site.yml:24]: Previous apps had this as .test instead of .stage.

**Comment by e-marie-w** [comment]
I would chalk the changes up to ACN, but I dug up the [uiowa07 setup PR](https://github.com/uiowa/uiowa/pull/7145/files) and it doesn't have these differences.

**Comment by joewhitsitt** [comment]
yeah the uiowa07 was a learning experience and had several additional PRs that followed the initial one

https://github.com/uiowa/uiowa/pull/7148
https://github.com/uiowa/uiowa/pull/7149
etc..

**Comment by e-marie-w** [comment]
> yeah the uiowa07 was a learning experience and had several additional PRs that followed the initial one

I see now, sorry for the false alarm!!

**Review by e-marie-w — APPROVED** [review]
I wasn't looking at the current drush file, my bad. The places I marked do match the patterns of our uiowa07 file so I am approving.

**Pull request merged**